synfig-studio/src/synfigapp/canvasinterface.cpp

```diff
@@ -1002,73 +996,75 @@ CanvasInterface::import_sequence(
 			String short_filename = CanvasFileNaming::make_short_filename(get_canvas()->get_file_name(), filename.u8string());
 			
 			try {
-				layer = add_layer_to("Import",get_canvas());
+				Layer::Handle layer = add_layer_to("Import", get_canvas());
 				if (!layer)
 					throw int();
 				if (!layer->set_param("filename", ValueBase(short_filename)))
 					throw int();
 
-				if(remove_dups){
-					curr_layer= Layer_Bitmap::Handle::cast_dynamic(layer);
-					if(!curr_layer){
+				bool is_layer_equal_previous_one = false;
+
+				if (remove_dups) {
+					// Gets the surface of current imported image
+					auto curr_layer = Layer_Bitmap::Handle::cast_dynamic(layer);
```

Comment:
Maybe I should change it to `static_cast`? The layer is Import type as defined in line 999.

Reply:
I suggest to leave it as-is.

Reply:
Ok